src/osha/oira/locales/bg/LC_MESSAGES/euphorie.po

```diff
@@ -23,6 +23,10 @@ msgstr ""
 msgid "${certificate/title}"
 msgstr ""
 
+#: osha.oira/src/osha/oira/client/browser/templates/dashboard-banner.pt:15
+msgid "${message/img_alt}"
+msgstr ""
+
```

Comment:
```suggestion
```
This shows an error in the template.

I will fix it, but I will not merge this bogus `msgid` for the moment.

Reply:
Removed here https://github.com/euphorie/osha.oira/pull/212